starter-code/pizza.js

```diff
@@ -1 +1,25 @@
 // Write your Pizza Builder JavaScript in this file.
+$('.btn-pepperonni').click(function (){
+  $('.pep').toggle();
+  $('.btn-pepperonni').toggleClass('active');
+});
+
+$('.btn-mushrooms').click(function (){
+  $('.mushroom').toggle();
+  $('.btn-mushrooms').toggleClass('active');
+});
+
+$('.btn-green-peppers').click(function (){
+  $('.green-pepper').toggle();
+  $('.btn-green-peppers').toggleClass('active');
+});
+
+$('.btn-crust').click(function (){
+  $('.crust').toggleClass("crust-gluten-free");
+  $('.btn-crust').toggleClass('active');
+});
+
+$('.btn-sauce').click(function (){
```

Comment:
```$(<selector>).click()``` works but it is preferable to use ```$(<selector>).on('click', function(){})``` as it will work with dynamically added elements as well